activestorage/lib/active_storage/attached/configuration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
This is the new configuration object that's yielded to the block. It's really just a ruby `Hash` that has setter/configuration methods on it. 

As mentioned in the PR description, this can be added to over time as `ActiveStorage` builds out other features that allow for configuration (if any)